ci/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Why not use the same package.json of the root and install all deps as devDeps?

Reply:
I consider them to be two different projects.

These deps are not related to greenworks, only to the CI and they are not usedthe same time nor the same way